build.cake

```diff
@@ -73,7 +74,7 @@ Task("BuildArtifacts")
     .IsDependentOn("Build")
     .Does(() =>
     {
-        foreach (var targetFramework in new[] { "net452", "netstandard1.5", "netstandard2.0" })
+        foreach (var targetFramework in new[] { "net452", "netstandard1.5", "netstandard2.0", "netstandard2.1" })
```

Comment:
Should be testing on .NET Core 3.x as well.

Reply:
Done

Reply:
I actually came back and removed the support for .NET Core 3 just to keep the PR in line with what we currently do.